pkg/artifactory/resource_artifactory_federated_repository.go

```diff
@@ -0,0 +1,277 @@
+package artifactory
+
+import (
+	"context"
+	"fmt"
+	"net/http"
```

Comment:
I'm not seeing where this gets used

Reply:
The `net/http` is used in line 200: `if resp.StatusCode == http.StatusNotFound {`